.github/workflows/check.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       _R_CHECK_SYSTEM_CLOCK_: 0
 
     container: 
-      image: pecan/depends:R${{ inputs.R-version }}
+      image: pecan/depends:develop-R${{ inputs.R-version }}
```

Comment:
Note that this means all check jobs pull from the `develop-*` depends image, even for checks not run against the develop branch (e.g. `latest`). This was true before too, just more obvious now that it shows up in the tag name.